package.json

```diff
@@ -47,6 +47,7 @@
     "@angular/platform-server": "16.2.0",
     "@nx/jest": "^16.10.0",
     "@nx/js": "^16.10.0",
+    "@types/jest": "^29.5.12",
```

Comment:
Had to manually hoist `jest` and `@types/jest` to the top level, where `@nx/jest` could use them. Previously, they were hoisted by `npm`, but after update they are no longer hoisted automatically